src/language/languageServerController.ts

```diff
@@ -143,15 +160,22 @@ export default class LanguageServerController {
     // Send a request with a cancellation token
     // to the language server instance to execute scripts from a playground
     // and return results to the playground controller when ready.
-    const result: ShellEvaluateResult = await this._client.sendRequest(
+    const res: ShellEvaluateResult = await this._client.sendRequest(
       ServerCommands.EXECUTE_CODE_FROM_PLAYGROUND,
       playgroundExecuteParameters,
       this._source.token
     );
 
     this._isExecutingInProgress = false;
 
-    return result;
+    log.info('Evaluate response', {
+      namespace: res?.result?.namespace,
+      type: res?.result?.type,
+      outputLength: JSON.stringify(res?.result?.content || '').length,
```

Comment:
Same with output. Maybe we receive something big that crashes a rendering of the response virtual editor.